x/retry/policyprovider.go

```diff
@@ -84,5 +90,8 @@ func (ppp *ProcedurePolicyProvider) Policy(_ context.Context, req *transport.Req
 	if pol, ok := ppp.serviceProcedureToPolicy[serviceProcedure{Service: req.Service}]; ok {
 		return pol
 	}
+	if pol, ok := ppp.serviceProcedureToPolicy[serviceProcedure{Procedure: req.Procedure}]; ok {
+		return pol
+	}
```

Comment:
Are these ordered such that the more common lookup will shadow the less common lookup? Do we have enough information to know this yet?